src/lessons/05_css_html/introduction_to_css.slides.html

```diff
@@ -0,0 +1,346 @@
+                <script type="text/template">
+                  <label for="username">Username</label>
+                  <input type="text" name="username" id="username">
+                  <label for="quantity">Quantity</label>
+                  <input type="number" name="quantity" id="quantity">
+              </script>
+            </code>
+            </pre>
+            <p class="fragment"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input" class="small">More on input element</a></p>
+          </section>
+          <section>
+            <h2>&lt;button&gt;</h2>
+            <ul>
+              <li class="fragment">Represents a clickable element</li>
+              <li class="fragment">It generally indicates that an action can by done by pressing it</li>
+              <li class="fragment">If the button is not for submitting a form we need to set their <span class="highlight">type</span> to <span class="highlight">button</span> </li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
+                <script type="text/template">
+                  <button>Submit!</button>
+                  <button type="Button">It does submit</button>
+              </script>
+            </code>
+            </pre>
+            <p class="fragment"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button" class="small">More on button element</a></p>
+          </section>
+          <section>
+            <h2>&lt;select&gt;</h2>
+            <ul>
+              <li class="fragment">It provides a menu of options</li>
+              <li class="fragment">Each option has a value represented by the <span class="highlight">&lt;select&gt;</span> tag</li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
+                <script type="text/template">
+                  <select name="housepets">
+                    <option value="cat">Cat</option>
+                    <option value="dog">Dog</option>
+                    <option value="llama">Llama</option>
+                  </select>
+              </script>
+            </code>
+            </pre>
+            <p class="fragment"><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select" class="small">More on select element</a></p>
+          </section>
+          <section>
+            <h2>&lt;textarea&gt;</h2>
+            <ul>
+              <li class="fragment">Represents a multiline text control</li>
+              <li class="fragment">Useful when we want to capture large amounts of text from the user</li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
+                <script type="text/template">
+                  <textarea name="textarea" rows="10" cols="50">
+                    Write something here
+                  </textarea>
+              </script>
+            </code>
```

Comment:
Apparently highlight.js isn't detecting this as HTML automatically. 
```suggestion
              <code class="fragment formattedCode html">
                <script type="text/template">
                  <textarea name="textarea" rows="10" cols="50">
                    Write something here
                  </textarea>
              </script>
            </code>
```